dnf-behave-tests/dnf/plugins-core/copr.feature

```diff
@@ -122,6 +123,10 @@ Given I create and substitute file "//etc/dnf/plugins/copr.conf" with
       protocol = http
       port = {context.dnf.ports[copr]}
       """
+  And HTTP server GET /{context.dnf.tempdir}/coprs/testuser/testproject/repo/fedora-31/dnf.repo response headers are
+      """
+      ('Copr-Error-Data', 'eyJhdmFpbGFibGUgY2hyb290cyI6IFsiZmVkb3JhLTMwLXg4Nl82NCJdfQ==')
```

Comment:
This is kind-of a moot question in this context, but why are you base64-encoding text data? That makes little sense. I guess it's because you'd need to escape some characters in it? Poor excuse. This sucks, because this base64 string is not human readable, anyone reading it will not know what's inside without decoding. Any change to this string will again need to be encoded and hence it will not be clear in the PR what the change is, the reviewer again needs to go and decode.